save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/TableComponent.kt

```diff
@@ -133,7 +133,7 @@ fun <D : Any> tableComponent(
         additionalOptions()
     }, plugins = plugins)
 
-    useEffect(arrayOf<dynamic>(tableInstance.state.pageSize, pageCount)) {
+    useEffect(tableInstance.state.pageSize) {
```

Comment:
pageCount is not needed?

Reply:
It created a circular dependency, i.e. we would query page count and update it whenever page count was changed. Also, page count depends on page size, so it will get updated whenever page size changes